.github/workflows/pre-submit.actions.yml

```diff
@@ -147,11 +147,11 @@ jobs:
         with:
           path: __BUILDER_CHECKOUT_DIR__
 
-      - name: Checkout the Go repository
+      - name: Checkout the JS repository
         uses: ./__BUILDER_CHECKOUT_DIR__/.github/actions/secure-project-checkout-node
         with:
           path: __PROJECT_CHECKOUT_DIR__
-          go-version: 16
+          node-version: 16
```

Comment:
Note to reviewers: This looks like it was a copy & paste miss. GitHub Actions doesn't seem to produce an error for missing required inputs

Reply:
Added #1907 to track.